network-api/networkapi/wagtailpages/pagemodels/blog/blog_index.py

```diff
@@ -119,6 +133,11 @@ class BlogIndexPage(IndexPage):
 
     template = 'wagtailpages/blog_index_page.html'
 
+    def get_context(self, request):
+        context = super().get_context(request)
```

Comment:
@tbrlpld  The reason for adding the related topics to the context is because it appears that design would like us to also add "related topics" to the actual topics themselves, and to render those on the topic filter pages.

I figured since this is a routable page, we can tap into the "topic" route and say "if we are filtering based on x topic, grab its "related topics" and update the context so we can render those in the component"

Reply:
Yup, makes sense. I like to be explicit about what I am adding to the context anyhow.